.nvmrc

```
@@ -0,0 +1 @@
+v18.12.1
```

I suggest:

```suggestion
20
lts/hydrogen
```

to use the newest version by default, and the latest Hydrogen (not a specific version) as a fallback